### ISSUE TITLE
Cut release for GHC 8.6.1

### ISSUE THREAD

**Issue by bgamari** [kolmodin/binary#149]
Thanks in advance!

**Comment by kolmodin** [comment]
I just went on vacation and will be away 2 weeks. Unfortunately I didn't
bring a laptop. I return the 27th of July. Can this wait?
Or, I should be able to accept PRs from the phone, if you're willing to do
the cut.
I usually just compare what changed from the last version, version bump,
check that the changelog is updated, make a tag, upload to hackage.

**Comment by bgamari** [comment]
Ping.

**Comment by kolmodin** [comment]
I just had a quick look, it looks like the next version needs have a major bump.

```diff
+#if __GLASGOW_HASKELL__ >= 800
+#define HAS_DATA_KIND
+#endif

-newtype Tagged (s :: * -> *) b = Tagged {unTagged :: b}
+#ifdef HAS_DATA_KIND
+newtype Tagged (s :: Type -> Type) b = Tagged {unTagged :: b}
+#else
+newtype Tagged (s :: * -> *)       b = Tagged {unTagged :: b}
+#endif
```

Since `0.8.5.1` was also released as `0.9.0.0` this new version needs to be `0.10.0.0`. I'll try to release it within a few days.

**Comment by kolmodin** [comment]
Done! Version 0.10.0.0 is out. Let me know how it works for you.
